bundle/manifests/fence-agents-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -1,17 +0,0 @@
-apiVersion: v1
```

Comment:
I think we want to keep this service?

Reply:
When I [changed the group](https://github.com/medik8s/fence-agents-remediation/pull/7/files#diff-d1fb402f1269ba7396dca9cbd03b44ac055874669bce1d0f239fa2f54ea3584bR15) for the CRDs, CSV and etc. I also deleted the old bundle directory so any old files would be deleted.
IIRC the bundle directory is created and updated after we run [`make bundle`](https://github.com/medik8s/fence-agents-remediation/blob/main/Makefile#L261).
After I deleted the bundle directory and created a new CRDs, CSV, and etc. then this service and `fence-agents-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml` wouldn't be created. Therefore, IIUC these files are redundant or I am missing something

Reply:
Oh, my bad, didn't see that they were redundant before, for whatever reason. 👍🏼